### ISSUE TITLE
Newsletter for WoT

### ISSUE THREAD

**Issue by egekorkan** [w3c/wot-marketing#156]
I had subscribed to the newsletter of AsyncAPI and got one issue of it (available [here](https://www.asyncapi.com/blog/february-2021-at-asyncapi?utm_source=newsletter&utm_medium=email&utm_content=button&utm_campaign=newsletter-16-03-2021)). This is also a standardization effort (but not under an SDO) and they share some updates quite nicely in a casual fashion. I think we can do something similar and would be a nice way to make people aware of the ongoing discussion in a curated way. The other way for someone to understand what is going on is going through GitHub issues and meeting minutes, both of which are not user-friendly for this purpose.

I am aware that this is some work but I wanted to create the issue anyways :) If someone were to create the content, we still need a way for people to subscribe to the newsletter. Is there a W3C solution for this @ashimura and @koalie ?

**Comment by koalie** [comment]
Hi Ege,

> On 16 Mar 2021, at 17:21 , Ege Korkan ***@***.***> wrote:
> 
> […]
> I am aware that this is some work but I wanted to create the issue anyways :) If someone were to create the content, we still need a way for people to subscribe to the newsletter. Is there a W3C solution for this @ashimura and @koalie ?

There isn’t one at the group level.
There is a W3C Newsletter that I send every week (usually on the Mondays but I forgot yesterday). 

It pulls existing content in W3C publications, events and announcements, spanning the last 6 days. W3C Blog posts are part of what is pull. So when the group gets to blogging (that is still outstanding), the post will be included the following Monday.

cf. <https://www.w3.org/News/Public/>
<https://lists.w3.org/Archives/Public/w3c-announce/> 

Coralie

--
Coralie Mercier  -  W3C Marketing & Communications -  https://www.w3.org
***@***.*** +337 810 795 22 https://www.w3.org/People/Coralie/

**Comment by egekorkan** [comment]
@koalie Thank you for the input! Then I have some follow-up questions ^^

We can simply write blog posts every month like a newsletter. However, if someone is specifically interested in the WoT work, how can they receive the emails? Possibilities ways:

1. Can we have a group-level newsletter functionality developed maybe in the future (by Studio24)  ? In order to not duplicate the writing effort, what we write for it becomes part of the W3C newsletter as well. 
2. Can one subscribe to the entire W3C Newsletter but select subtopics?
3. Are we allowed to create our own solution?

I went to the newsletter page but the subscribe link does not seem to work. Also, it is a bit difficult to find that text (just a feedback :blush: )

**Comment by danielpeintner** [comment]
I don't have any experience with it but I wonder whether it is possible to allow people to subscribe to certain topics via GitHub and let them know by means of GitHub about any news?

https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github

**Comment by koalie** [comment]
Hi Ege,

Reading your response made me realise my own response was incomplete! I wanted to suggest that the group is free to use a separate system (I was going to recommend MailChimp) for your own dedicated newsletter. I am happy to give you some basic guidelines, DOs and DONTs, to help with the template layout and branding, review the first few newsletters and then let you be on your own without oversight.

Please see inline.

> On 16 Mar 2021, at 19:38 , Ege Korkan ***@***.***> wrote:
> 
> 
> @koalie Thank you for the input! Then I have some follow-up questions ^^
> 
> We can simply write blog posts every month like a newsletter.

I really thing the group ought to write blog posts when it needs to, as opposed to when it wants to be featured in the W3C newsletter.

> However, if someone is specifically interested in the WoT work, how can they receive the emails? Possibilities ways:
> 
> 	• Can we have a group-level newsletter functionality developed maybe in the future (by Studio24) ? In order to not duplicate the writing effort, what we write for it becomes part of the W3C newsletter as well.

-1 unfortunately to Studio 24 being in the loop; our contract with them was very specific and any newsletter is out of scope (not even the W3C’s). Integrating with the W3C newsletter seems not easy to do either. 

But yes, as I mentioned at the start, you can have a group-level endeavour.

> 	• Can one subscribe to the entire W3C Newsletter but select subtopics?

Nope.

> 	• Are we allowed to create our own solution?

Yes, but w3c MarComm needs to be looped-in [at best to help you, at worst to guarantee you’re not doing something that would then require us to have to manage any crisis].

> I went to the newsletter page but the subscribe link does not seem to work.

It works for me. Clicking “subscribe” opens a pre-filled e-mail. Then you need to click send.

> Also, it is a bit difficult to find that text (just a feedback 😊 )

I use (ctrl|cmd)+s to search any page for what I’m after, so that too worked for me :D

But point taken, it doesn’t have to be hidden at the very bottom and it should be a button, not a link.

Coralie

--
Coralie Mercier  -  W3C Marketing & Communications -  https://www.w3.org
***@***.*** +337 810 795 22 https://www.w3.org/People/Coralie/

**Comment by egekorkan** [comment]
From the call of 11.01.2022:

We want to list our requirements for the Mar Comm team and ask for possible solutions before going with our own approach:
1. Interested people get information into their own email inbox
2. The information is WoT specific
3. (optional) Archived content on a web page
4. Interested people should be able to subscribe and unsubscribe (periodic reminders to allow unsubscription)
5. Posting is not permitted by anyone, only allowed for editors and reviewers of the WoT Marketing TF.

@koalie is there any W3C tool that meets these requirements? W3C Newsletter does not meet 2, blog post does not meet 1.

**Comment by koalie** [comment]
I will run this list of requirements by my colleagues and will let you know as soon as we can our feedback.
Coralie

> On 11 Jan 2022, at 13:59 , Ege Korkan ***@***.***> wrote:
> 
> 
> From the call of 11.01.2022:
> 
> We want to list our requirements for the Mar Comm team and ask for possible solutions before going with our own approach:
> 
> 	• Interested people get information into their own email inbox
> 	• The information is WoT specific
> 	• (optional) Archived content on a web page
> 	• Interested people should be able to subscribe and unsubscribe (periodic reminders to allow unsubscription)
> 	• Posting is not permitted by anyone, only allowed for editors and reviewers of the WoT Marketing TF.
> @koalie is there any W3C tool that meets these requirements? W3C Newsletter does not meet 2, blog post does not meet 1.
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android. 
> You are receiving this because you were mentioned.
> 

--
Coralie Mercier  -  W3C Marketing & Communications -  https://www.w3.org
***@***.*** +337 810 795 22 https://www.w3.org/People/Coralie/

**Comment by egekorkan** [comment]
Regarding CG or IG, in the call of 12.07, we said that it would depend on the content, i.e. if the newsletter is about WoT news it can be CG, if it is about new specification versions, it can be IG.

If it is from IG, W3C needs to be represented

**Comment by koalie** [comment]
Hi @egekorkan, all,

I needed to get back to you on the Newsletter topic. My apologies for the delay. We knew about half of the topic and resolved of the rest in the last week or so:

* The [W3C weekly newsletter](https://www.w3.org/News/Public/) as we know it will be discontinued when we transition to the redesign W3C website. 
* However, the new website will feature W3C news and W3C Blog posts in the same area, and there will be an RSS feed (To be confirmed: what granularity can we offer? e.g. if there's a tag "wot" for example, can this be "subscribed to"?)
* Given that doing today's newsletter relies on existing files which will no longer exist in the new website, generation of the newsletter going forward can no longer be done via xslt and unix scripts in a few minutes as is the case today. Given that the people-power in the MarComm team is insufficient already and that adopting a new newsletter solution will be more time-consuming, I have resolved NOT to pursue a W3C newsletter in the future W3C website for the time being. We keep this as a project to consider for the future once we are in a position to to a good job of it.

**Comment by egekorkan** [comment]
Thank you @koalie for the update. In this case, it seems that we should organize a solution ourselves right? If the MarComm team has more people-power in the future, we would be happy to migrate there if we can copy over the subscribed email addresses.

**Comment by koalie** [comment]
@egekorkan yes, I suppose. Please refer back to my earlier note: I'd like for w3c MarComm to be looped-in for branding questions at least, possibly other aspects.
